treeherder/auth/backends.py

```diff
@@ -51,30 +51,6 @@ def _extract_email_from_clientid(self, client_id):
         raise NoEmailException(
             "No email found in clientId: '{}'".format(client_id))
 
-    def _find_user_by_email(self, email, username, scopes):
-        """
-        Try to find an existing user that matches the email.
-        """
-
-        if scopeMatch(scopes, [["assume:mozilla-user:{}".format(email)]]):
```

Comment:
With this `scopeMatch` reference removed, the import is now unused. Could you remove it to fix the flake8 lint Travis failure? (There's also another unused import in test_backends.py, which is cause flake8 to fail too)